Makefile

```diff
@@ -3,7 +3,7 @@ APP_ROOT := $(CURDIR)
 APP_NAME := raven
 
 # Anaconda
-ANACONDA_HOME ?= $(HOME)/miniconda
+ANACONDA_HOME ?= $(HOME)/anaconda3
```

Comment:
I'm concerned this will break other people's code.

Reply:
Yes I had to do this to run on my machine, I'll remove it.

Reply:
I think you can set your own environment variable to override this default.